src/apps/testapps/testCompact.c

```diff
@@ -19,10 +19,10 @@
 #include "h3Index.h"
 #include "test.h"
 
-H3Index sunnyvale = 0x89283470c27ffffl;
+H3Index sunnyvale = 0x89283470c27ffff;
```

Comment:
Removing the long indicator on this constant will get you a different number (at least the last time I tested this it did). Try putting a `printf` statement with the H3Index in it and see if the number is the same.

Reply:
Really? I didn't have any issue here:

Source:
```
#include <inttypes.h>
#include <stdint.h>
#include <stdio.h>

int main(void) {
  uint64_t x = 0x89283470c27ffff;
  uint64_t y = 0x89283470c27ffffl;
  printf("0x%" PRIx64 ", 0x%" PRIx64 "\n", x, y);
  return 0;
}
```
Output:
`0x89283470c27ffff, 0x89283470c27ffff`

Reply:
I get the same results as @isaachier here, and I'm pretty sure the tests wouldn't pass if these weren't being interpreted correctly. Leaving with no suffix for now.